packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -349,8 +351,12 @@ async function makeE2ETasks () {
       return null
     },
 
-    __internal_remoteGraphQLIntercept (fn: string) {
-      remoteGraphQLIntercept = new Function('console', 'obj', 'testState', `return (${fn})(obj, testState)`).bind(null, console) as RemoteGraphQLInterceptor
+    __internal_remoteGraphQLIntercept (args: {
```

Comment:
This is so we can pass data from the frontend to `remoteGraphQLIntercept`

That's what I do here https://github.com/cypress-io/cypress/pull/25601/files#diff-b0772ecf528a07f437bcf6dc261b015b048d3403c7cecc62a3b18823cc891a37R28